terms.html

```diff
@@ -83,7 +83,7 @@
 distribution or use of identity information can compromise privacy. Collection
 and use of such information should follow the principles of minimal disclosure.
   </dd>
-  <dt><dfn data-lt="identifier registries">identifier registry</dfn></dt>
+  <dt><dfn data-lt="verifiable data registries">Verifiable data registry</dfn></dt>
```

Comment:
This definition will need to be expanded based on the expansion of scope above.

Reply:
@brentzundel could you take at improving this definition?

Reply:
done